components/region_cache_memory_engine/src/write_batch.rs

```diff
@@ -94,6 +97,46 @@ impl RangeCacheWriteBatch {
         }
     }
 
+    /// Trigger a CleanLockTombstone task if the accumulated lock cf
+    /// modification exceeds the threshold (16MB).
+    ///
+    /// NB: It need to acquire RocksDB mutex to get oldest snapshot, so do not
+    ///     call it on any RocksDB's callback, e.g., write batch callback.
```

Comment:
```suggestion
    /// NB: Acquiring the RocksDB mutex is necessary to get the oldest snapshot,
    ///     so avoid calling this in any RocksDB callback (e.g., write batch
    ///     callback) to prevent potential deadlocks.
```